Makefile

```diff
@@ -118,38 +144,60 @@ CROSS_BUILD_TARGETS := \
 	bin/podman.cross.linux.mips64 \
 	bin/podman.cross.linux.mips64le
 
-.PHONY: all
-all: binaries docs
```

Comment:
I strongly object to this change. No human being in the history of the world has wanted `make` to emit a help message. `make` should make. I've been typing `make` in this repo to build podman; I imagine that pretty much everyone who has ever built podman is also used to this. 

Please leave `all` as the default target. Anyone who wants help can type `make help`.

Reply:
I'm fine either way, the only reason I made this change was because the literal target "default" was not the actual default.  If this is the consensus, I'm happy to simply remove the target called "default".